rero_ils/modules/items/tasks.py

```diff
@@ -64,7 +64,7 @@ def process_late_claimed_issues(
             create_first_claim_count=create_first_claim_count,
             create_next_claim_count=create_next_claim_count
         )
-
+    set_time_stamp('claims-creation', msg=msg)
```

Comment:
is this function couldn't be a decorator?
something like :
```python
def set_timestamp(func):
    @wraps(func)
    def wrapped(*args, **kwargs):
        result = func(*args, **kwargs)
        # do the job to log the timestamp here with `func.__name__` as parameter 
        return result 
    return wrapped
```

Reply:
Will add an additionally function wrapper.